networking/p2p/src/main/java/tech/pegasys/teku/networking/p2p/discovery/DiscoveryNetwork.java

```diff
@@ -105,6 +107,18 @@ public Optional<String> getEnr() {
     return discoveryService.getEnr();
   }
 
+  @Override
+  public Optional<UInt256> getDiscoveryNodeId() {
```

Comment:
Could you please also add a test checking that UInt256 will match BigInteger with 256bit and smaller numbers. somthing with when(discoveryService.getNodeId).then return new(BigInteger("12345)) -> UInt256.toString match 12345

Reply:
Done